core/controllers/payload_validator_test.py

```diff
@@ -0,0 +1,86 @@
+# Tests.
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for various errors raised by validate method of payload validator."""
+
+from __future__ import absolute_import  # pylint: disable=import-only-modules
+from __future__ import unicode_literals  # pylint: disable=import-only-modules
+
+from core.controllers import payload_validator
+from core.tests import test_utils
+
+
+class ErrorValidationUnitTests(test_utils.GenericTestBase):
+
+    def test_invalid_args_raises_exceptions(self):
+        list_of_invalid_args_with_schema_and_errors = [
```

Comment:
Add a comment stating what each element in the list comprises of.

Reply:
Done